packages/studio-plugin/src/parsers/StudioSourceFileParser.ts

```diff
@@ -101,11 +101,21 @@ export default class StudioSourceFileParser {
 
   parseCssImports(): string[] {
     const cssImports: string[] = [];
+    const getAbsolutePath = (source: string) => {
+      if (source.startsWith(".")) {
+        const absPath = upath.join(
```

Comment:
is there a way to use `upath.resolve` instead of `join` here? then we can likely remove the check on line 105 and I think it would make it a little more clear that we're definitely returning an absolute path

Reply:
used path.resolve!